conf/muchos.props.example

```diff
@@ -46,6 +46,10 @@ accumulo_version = 2.0.0-alpha-1
 download_software = True
 # Install Hub (for GitHub)
 install_hub = True
+# The java package to install
+java_package=java-1.8.0-openjdk-devel
+# The package to use for java 11
+# java_package=java-11-openjdk-devel
```

Comment:
Should 11 be the default now?

Reply:
Last time I tried using java Hadoop Yarn would not start.

Reply:
I fixed that in Uno. Perhaps you can do something similar to https://github.com/apache/fluo-uno/commit/c3212c5b75b93a81830d22a3e6bdf0155ce1f2f1 ?

Reply:
Can be a separate issue.

Reply:
That may be good to do.  I am not too familiar with ansible, so it would not be quick for me.

Reply:
#266